README.md

```diff
@@ -14,6 +14,7 @@ RSDKUtils consists of 4 sub-modules:
 
 This module supports iOS 12.0 and above. It has been tested with iOS 12.0 and above.
 This module also supports watchOS 6.0 and above, and it's compatible with extensions.
+This module also supports macOS 11.0 and above.
```

Comment:
Maybe let's combine those 3 sentences about supported OSes. I'm not good at it but maybe something like:

This module supports following operating systems:
* iOS 12.0 and above (including extension contexts)
* watchOS 6.0 and above
* macOS 11.0 and above

It has been tested with iOS 12.0 and above.